hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -168,10 +172,13 @@ objects:
       managed.openshift.io/gitHash: ${IMAGE_TAG}
       managed.openshift.io/gitRepoName: ${REPO_NAME}
       managed.openshift.io/osd: 'true'
-    name: aws-vpce-operator-hypershift-sss
+    name: aws-vpce-operator-hypershift-sss-${SECTOR}
   spec:
     clusterDeploymentSelector:
       matchExpressions:
+        - api.openshift.com/legal-entity-id: "${OCM_ORG}"
+        - ext-openshift-aws-vpce-operator/cluster: "true"
+        - ext-openshift-aws-vpce-operator/sector: "${SECTOR}"
```

Comment:
I am not aware of the existence of those 2 `ext-openshift-aws-vpce-operator/*` labels. Which component sets them?

Reply:
As discussed, let's use the following labels:
```
hive.openshift.io/cluster-region: "${REGION}" <-- this should be available for all ClusterDeployments for all OSD operators
ext-hypershift.openshift.io/cluster-sector: "${SECTOR_NAME}"   <-- this is hypershift specific
```